lms/djangoapps/courseware/views/views.py

```diff
@@ -980,6 +981,15 @@ def _progress(request, course_key, student_id):
     # checking certificate generation configuration
     enrollment_mode, _ = CourseEnrollment.enrollment_mode_for_user(student, course_key)
 
+    user_role = CourseAccessRole.objects.filter(
+        user=request.user,
+        course_id=course_key
+    ).values_list('role', flat=True)
+
+    is_local_staff = True
```

Comment:
Please describe in the comment what does local_staff mean?

Reply:
Done